u8timeseries/custom_logging.py

```diff
@@ -16,10 +16,10 @@ def get_logger(name):
     """
     logger = logging.getLogger(name)
     logger.setLevel(logging.INFO)
-    stderr_handler = logging.StreamHandler()
+    handler = logging.StreamHandler()
```

Comment:
How about putting handler as parameter?

Reply:
And set logging.StreamHandler() as its default value?